page-h5p-listing.php

```diff
@@ -3,40 +3,54 @@
 <?php
 if ( \PressbooksBook\Helpers\is_book_public() ) :
 	$data = \PressbooksBook\Helpers\get_h5p_activities();
+	$activities = count( $data['activities'] );
 	?>
 	<div id="post-<?php the_ID(); ?>" <?php post_class( 'h5p-listing-page' ); ?>>
-			<h1 class="page-title"><?php echo __( 'H5P activities list', 'pressbooks-book' ); ?></h1>
+		<h1 class="page-title">
+			<?php
+			echo isset( $_GET['h5p_id'] ) ?
```

Comment:
Probably it could be better to check for isset GET parameter and also the activities count because if the parameter is empty or we pass an inexistent value we are going to get 
![image](https://user-images.githubusercontent.com/4997549/145251956-f8c8bb9b-a474-4244-a0a7-117155af5fea.png)

Reply:
thank you for the suggestion bro. Yes, definitely the invalid H5P activities must be handled. I already pushed some changes considering that case.